docker/mongodb/startup-script-mongo/initialize-mongo-rs.sh

```diff
@@ -2,7 +2,7 @@
 
 echo "Executing initialize-mongo-rs.sh"
 
-mongo --eval "mongodb = ['$NODE_HOSTNAME:32001', '$NODE_HOSTNAME:32002', '$NODE_HOSTNAME:32003'], rsname = '$RS_NAME'" --shell << EOL
+mongo --eval "mongodb = ['$NODE_HOSTNAME_ONE:37017', '$NODE_HOSTNAME_TWO:37017', '$NODE_HOSTNAME_THREE:32017'], rsname = '$RS_NAME'" --shell << EOL
```

Comment:
@arooshap 

Are you sure about ports? `37017` in node 1 and 2 and `32017` in node 3? why different ports?

Reply:
@muhammadimranfarooqi I am still making changes to this.